### PR TITLE
Refactor language parameter mapping in install_from_release.sh

### PR DESCRIPTION
**Pull request by naioja**
Refactor language parameter mapping in install_from_release.sh for all of translation we offer, including a catch all option.

### CODE REVIEW

File: scripts/install_from_release.sh [hunk 1]
```diff
@@ -12,13 +12,42 @@ fi
...
+  fi
+elif [ "$MAIN_LANG_PREFIX" = "zh" ]; then
+  if [ "$LANG" = "zh-cn" ]; then
+    SCENARIOS="https://github.com/MicrosoftDocs/executable-docs/releases/download/v1.0.1/zh-cn-scenarios.zip"
+  elif [ "$LANG" = "zh-tw" ]; then 
+    SCENARIOS="https://github.com/MicrosoftDocs/executable-docs/releases/download/v1.0.1/zh-tw-scenarios.zip"
+  else
+    SCENARIOS="https://github.com/MicrosoftDocs/executable-docs/releases/download/v1.0.1/zh-cn-scenarios.zip"
+  fi
+elif [[ ${LANG_ARRAY_1[@]} =~ $MAIN_LANG_PREFIX ]]; then
```
Comment (vmarcella) [inline scripts/install_from_release.sh:35]: Can we surround these variable accesses with quotations?

File: scripts/install_from_release.sh [hunk 2]
```diff
@@ -12,13 +12,42 @@ fi
...
   SCENARIOS="https://github.com/Azure/InnovationEngine/releases/download/$RELEASE/scenarios.zip"
-# Otherwise, download the scenarios from Microsoft Docs in the appropriate langauge
-elif [ "$LANG" = "en-us" ]; then
-  SCENARIOS='https://github.com/MicrosoftDocs/executable-docs/releases/download/v1.0.1/scenarios.zip'
+elif [ "$MAIN_LANG_PREFIX" = "pt" ]; then
+  if [ "$LANG" = 'pt-pt' ]; then
+    SCENARIOS="https://github.com/MicrosoftDocs/executable-docs/releases/download/v1.0.1/pt-pt-scenarios.zip"
+  elif [ "$LANG" = 'pt-br' ]; then
+    SCENARIOS="https://github.com/MicrosoftDocs/executable-docs/releases/download/v1.0.1/pt-br-scenarios.zip"
+  fi
```
Comment (mbifeld) [inline scripts/install_from_release.sh:26]: We need an else case here to download pt-pt incase the code is something else 'pt-xx'
```suggestion
  else
    SCENARIOS="https://github.com/MicrosoftDocs/executable-docs/releases/download/v1.0.1/pt-pt-scenarios.zip"
  fi
```

File: scripts/install_from_release.sh [hunk 3]
```diff
@@ -12,13 +12,42 @@ fi
...
+  elif [ "$MAIN_LANG_PREFIX" = "ja" ]; then
+    SCENARIOS="https://github.com/MicrosoftDocs/executable-docs/releases/download/v1.0.1/$MAIN_LANG_PREFIX-jp-scenarios.zip"
+  elif [ "$MAIN_LANG_PREFIX" = "ko" ]; then
+    SCENARIOS="https://github.com/MicrosoftDocs/executable-docs/releases/download/v1.0.1/$MAIN_LANG_PREFIX-kr-scenarios.zip"
+  elif [ "$MAIN_LANG_PREFIX" = "sv" ]; then
+    SCENARIOS="https://github.com/MicrosoftDocs/executable-docs/releases/download/v1.0.1/$MAIN_LANG_PREFIX-se-scenarios.zip"
+  fi
 else
-  SCENARIOS="https://github.com/MicrosoftDocs/executable-docs/releases/download/v1.0.1/$LANG-scenarios.zip"
+  SCENARIOS="https://github.com/Azure/InnovationEngine/releases/download/$RELEASE/scenarios.zip"
```
Comment (mbifeld) [inline scripts/install_from_release.sh:50]: I feel like we could move this last else to the beginning so it runs always (and remove 19-20).  Then if any of the if statements happen it'll just overwrite SCENARIOS.
Also in the default/english case, we're not downloading scenarios from IE anymore:
```suggestion
  SCENARIOS="https://github.com/MicrosoftDocs/executable-docs/releases/download/v1.0.1/scenarios.zip"
```

**Review by mbifeld — CHANGES_REQUESTED** [review]
Thanks so much for the fix Adrian!  Added a few comments

**Review by mbifeld — APPROVED** [review]

**Pull request merged**